client/app/license/order/license-order.html

```diff
@@ -194,75 +194,73 @@ <h2 data-translate="license_order_step3_title"></h2>
+                                        data-translate="license_order_step4_contracts_agora_ask">
+                                </strong>
+                                <span data-ng-repeat="contract in durations.details[selected.duration].contracts track by $id">
+                                    <a target="_blank"
+                                       rel="noopener"
+                                       class="d-inline-block oui-link_icon"
+                                       data-ng-href="{{ ::contract.url }}">
+                                       <span data-translate="common_newtab" class="sr-only"></span>
+                                       <span data-ng-bind="contract.url"></span>
+                                       <span class="oui-icon oui-icon-external_link" aria-hidden="true"></span>
+                                     </a>
+                                </span>
+                                <div class="checkbox">
+                                    <label>
+                                        <input type="checkbox"
+                                               data-ng-model="contractsValidated.value" />
+                                        <span data-translate="license_order_step4_contracts_check"></span>
+                                    </label>
+                                </div>
                             </div>
                         </div>
                     </div>
-
-                    <div class="col-md-3"
-                         data-ng-if="!agoraEnabled">
-                        <button type="button"
-                                class="btn btn-primary"
-                                data-ng-hide="loaders.bc"
-                                data-ng-click="generateBc()">
-                            <i class="icon-order-licence" aria-hidden="true"></i>
-                            {{ 'license_order_step4_generate' | translate }}
-                        </button>
-                        <span data-ng-show="loaders.bc"
-                              data-translate="license_order_step4_generating">
-                        </span>
-                        <oui-spinner data-ng-show="loaders.bc"
-                                     data-size="s">
-                        </oui-spinner>
-                    </div>
-                    <div class="col-md-3"
-                         data-ng-if="agoraEnabled">
-                        <a href="{{ selected.agoraUrl }}"
-                           class="btn btn-primary"
-                           target="_blank"
-                           data-ng-if="selected.agoraUrl"
-                           data-ng-hide="loaders.bc">
-                            <i class="icon-order-licence" style="margin-top: 5px !important"></i>
-                            {{ 'license_order_step4_pay_button' | translate }}
-                        </a>
+                    <div class="d-block">
```

Comment:
```suggestion
                    <div>
```
`div` is already a `block` 😄